CHANGELOG.md

```diff
@@ -35,6 +35,9 @@
 
 - [#5383](https://github.com/ChainSafe/forest/pull/5383) Add support for `Filecoin.EthGetFilterChanges` RPC method.
 
+- [#4505](https://github.com/ChainSafe/forest/issues/4505) The Ethereum RPC API indexer now runs as a background task. It is configurable in the `[chain_indexer]` section or via the 
+  `FOREST_CHAIN_INDEXER_ENABLED` environment variable. Moved `client.eth_mapping_ttl` to `chain_indexer.gc_retention_epochs`, which is now specified as a number of epochs.
```

Comment:
This is a breaking change, especially given it's now disabled by default. Let's mention it.